app/manifest.cue

```diff
@@ -78,6 +87,25 @@ manifest: versions: v1alpha1: {
 				// accessKinds is a list of KindPermission objects for accessing additional kinds provided by other apps
 				accessKinds: [...#KindPermission]
 			}
+			// DryRunKinds dictates whether this revision should create/update CRD's from the provided kinds,
+			// Or simply validate and report errors in status.resources.crds.
+			// If dryRunKinds is true, CRD change validation will be skipped on ingress and reported in status instead.
+			// Even if no validation errors exist, CRDs will not be created or updated for a revision with dryRunKinds=true.
+			dryRunKinds?: bool | *false
```

Comment:
I am debating whether this is better as-is, or if we want to add the ability to dry-run any resource types created/updated by the controller with the same structure as `status.resources`, like:
```cue
dryRun?: {
    [string]: bool
}
```
Or perhaps
```cue
dryRun?: {
    crds: bool | *false
    [string]: bool
}
```
Thoughts on this?

Reply:
Not a strong opinion. I feel this is fine as is, more predictable and should be fine for most use cases I can think of.

Reply:
IMHO kinds is fine for now.